### PR TITLE
Fix signal send for Gunicorn instead of for Apache

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: doc/developer/debugging.rst [hunk 1]
```diff
@@ -85,7 +85,7 @@ Edit a file in a running apache WSGI container
 
    docker-compose exec geoportal bash
    vi ...
-   kill -s USR1 1  # graceful
+   kill -s HUP `ps aux|grep gunicorn|head --lines=1|awk '{print $2}'`  # graceful
```
Comment (pvalsecc) [inline doc/developer/debugging.rst]: Then, the title is wrong...

Why don't you suggest to put gunicorn in devel mode (the --reload option) instead?

File: doc/developer/debugging.rst [hunk 2]
```diff
@@ -78,14 +78,20 @@ Actually we display the running rule and why she is running (dependence update).
 Docker
 ------
 
-Edit a file in a running apache WSGI container
+Run gunicorn to reload on modified python file
```
Comment (jwkaltz) [inline doc/developer/debugging.rst]: ... to reload on modifications of python files

File: doc/developer/debugging.rst [hunk 3]
```diff
@@ -78,14 +78,20 @@ Actually we display the running rule and why she is running (dependence update).
...
 ..............................................
 
+Add the following environement variable to the geoportal container:
```
Comment (jwkaltz) [inline doc/developer/debugging.rst]: environment

File: doc/developer/debugging.rst [hunk 4]
```diff
@@ -78,14 +78,20 @@ Actually we display the running rule and why she is running (dependence update).
...
+
+``GUNICORN_PARAMS="-b :80 --worker-class gthread --threads 1 --workers 1 --reload"``
+
+Gaceful the running geoportl container
```
Comment (jwkaltz) [inline doc/developer/debugging.rst]: Do a graceful restart of the running geoportal container

Comment (sbrunner) [reply doc/developer/debugging.rst]: Thanks, fixed :-)

**Pull request merged**